html/semantics/interactive-elements/the-dialog-element/dialog-requestclose.html

```diff
@@ -35,14 +35,8 @@
   assert_true(dialog.matches(':open'));
   assert_equals(dialog.matches(':modal'),openMethod === 'modal');
 }
-function getSignal(t) {
-  const controller = new AbortController();
-  const signal = controller.signal;
-  t.add_cleanup(() => controller.abort());
-  return signal;
-}
 
-async function setup(t,closedby) {
```

Comment:
The tests now don't need a `waitForRender` inside of `setup`, which means `setup` doesn't need to be `async`, which means none of the tests need to `await setup`.